jax/lax/lax.py

```diff
@@ -868,8 +868,8 @@ def scatter_add(operand: Array, scatter_indices: Array, updates: Array,
       relate.
     indices_are_sorted: whether `scatter_indices` is known to be sorted. If
       true, may improve performance on some backends.
-    unique_indices: whether `scatter_indices` is known to be free of duplicates.
-      If true, may improve performance on some backends.
+    unique_indices: whether the indices to be updated in operand are guaranteed to
```

Comment:
You should probably say ``operand`` to get sphinx to highlight it properly.

Reply:
Addressed in `eb2d7df4`